tests/PHPStan/Rules/Classes/EnumSanityRuleTest.php

```diff
@@ -14,7 +14,7 @@ class EnumSanityRuleTest extends RuleTestCase
 
 	protected function getRule(): Rule
 	{
-		return new EnumSanityRule();
+		return new EnumSanityRule($this->createReflectionProvider());
 	}
 
 	public function testRule(): void
```

Comment:
I wonder why this test runs on php8.. shouldn't it skip php 8.0 and requre 8.1+?

Reply:
//cc @kukulich who initally created this test

Reply:
Let it run on PHP 8, there's no reason not to if it doesn't crash.

The reason why 8.0+ is that ReflectionEnum stubs are 8.0+ because of the static return type.

If it works then keep it working. In theory it lets someone with 8.0 runtime set 80100 in phpVersionId and it should work.